src/data/add_metadata.py

```diff
@@ -37,13 +37,15 @@ def base_document_to_gst_document(
     new_metadata_dict["validation_status"] = "validated"
     new_metadata_dict["version"] = new_metadata_dict.pop("Document Role")
     new_metadata_dict["date"] = new_metadata_dict.pop("Date")
-    new_metadata_dict["link"] = new_metadata_dict.pop("Documents")
+    new_metadata_dict["link"] = new_metadata_dict.get("Documents")
     new_metadata_dict["document_variant"] = new_metadata_dict.pop("Document Variant")
     new_metadata_dict["author_is_party"] = new_metadata_dict["Author Type"] == "Party"
     new_metadata_dict["family_id"] = new_metadata_dict.pop("CPR Family ID")
     new_metadata_dict["family_slug"] = new_metadata_dict.pop("CPR Family Slug")
     new_metadata_dict["family_name"] = new_metadata_dict.pop("Family Name")
     new_metadata_dict["geography_iso"] = new_metadata_dict.pop("Geography ISO")
+    # NOTE: the document URL is also stored in the JSON, but is populated with "https://example.com" when the standalone parser is used to process just a PDF
```

Comment:
Not sure whether this is important here but I don't think we are doing this! 

https://github.com/climatepolicyradar/data-access/blob/main/src/cpr_data_access/parser_models.py

The BackendDocument and ParserInput types have optional AnyHttpUrl for source url and the azure pdf parser run.py doesn't appear to introduce example.com for source url.

Reply:
annoyingly i can't find the version of the parser i used to parse the docs for this as the parser wouldn't install with poetry, so will just remove the comment for now. i still think it's worth pulling the source url from the CSV so will leave that line in